pages/common/github-label-sync.md

```diff
@@ -0,0 +1,24 @@
+# github-label-sync
+
+> A command line interface for synchronising GitHub labels.
+> Homepage: <https://npmjs.com/package/github-label-sync>.
+
+- Synchronise labels using a local `labels.json` file:
+
+`github-label-sync --access-token {{token}} {{repository_name}}`
+
+- Synchronise labels using a specific labels JSON file:
+
+`github-label-sync --access-token {{token}} --labels {{url|path/to/json_file}} {{repository_name}}`
+
+- Don't actually synchronise labels (i.e. do a dry run):
```

Comment:
I am away from laptop atm. But how do we usually write dry run descriptions for other pages ? IMO mentioning the dry run part before may be better - Perform a dry run instead of actually synchronising labels.

Reply:
We have a few ways looking at previous pages:
- Dry-run - ...
- Perform a dry-run...
- Do a dry-run...

It does sound better with `Perform a dry run` as you said. 👍

Is it worth standardising this across pages? Or not really?

Reply:
I think it would be more intuitive to explicitly say something like "Simulate action X" for dry-run actions (I'm not opposed to actually using the "dry-run" term in addition to that wording).

Reply:
D'oh. 😋 I should leave things before committing. I'll wait until we decide on the wording.